README.md

```diff
@@ -6,13 +6,13 @@
 |   :--:  |    :---:      |   :---:  |       :---:      |       :---:       |      :---:      |
 |  v0.10  |**End-of-Life**|    -     |        -         |    2015-10-01     |   2016-10-31    |
 |  v0.12  |**End-of-Life**|    -     |        -         |    2016-04-01     |   2016-12-31    |
```

Comment:
While you're here, is there a reason these are `v0.10` and `v0.12` instead of `0.10.x` and `0.12.x`?